src/libpriv/rpmostree-core.cxx

```diff
@@ -842,6 +842,9 @@ rpmostree_context_setup (RpmOstreeContext    *self,
                                    DNF_TRANSACTION_FLAG_NODOCS);
   }
 
+  /* Force disable internal libdnf Count Me logic */
+  dnf_conf_add_setopt("*.countme", DNF_CONF_COMMANDLINE, "false", NULL);
```

Comment:
Minor/optional: should this be in `rpmostree_context_new_system` instead? This is where we disable other global-ish librpm/libdnf bits we know we never want upfront.